apps/ensapi/src/handlers/ensanalytics-api.ts

```diff
@@ -89,4 +94,81 @@ const app = factory
     }
   });
 
+// Referrer address parameter schema
+const referrerAddressSchema = z.object({
+  referrer: makeLowercaseAddressSchema("Referrer address"),
+});
+
+// Get referrer detail for a specific address
+app.get("/referrer/:referrer", validate("param", referrerAddressSchema), async (c) => {
```

Comment:
```suggestion
app.get("/referrers/:referrer", validate("param", referrerAddressSchema), async (c) => {
```

Goal: Align with the existing API endpoint. One is for paginating through all referrers, then if you add this extra detail in the path it ideally should give the details of one specific referrer.